webapp/platform/client/src/websocket.ts

```diff
@@ -231,11 +236,73 @@ export default class WebSocketClient {
             this.reconnectTimeout = setTimeout(
                 () => {
                     this.reconnectTimeout = null;
-                    this.initialize(this.connectionUrl, token, this.postedAck);
+                    this.initialize(connectionUrl, token, this.postedAck);
                 },
                 retryTime,
             );
         };
+        this.conn.onclose = onclose;
+
+        this.conn.onopen = () => {
```

Comment:
Is there any strong reason to move this function after `onclose`? It used to be before it, and the diff seems larger because of this change, making also a bit more difficult to understand what exactly changed.